packages/annotation-toolkit/src/utils.js

```diff
@@ -0,0 +1,5 @@
+export function isFunction(functionToCheck) {
+  return (
+    functionToCheck && {}.toString.call(functionToCheck) === "[object Function]"
```

Comment:
Wow, I'm surprised that's how you detect a function. 

Curious if the second answer from here also works? https://stackoverflow.com/questions/5999998/check-if-a-variable-is-of-function-type

Reply:
I think I've used that before:
```
if (typeof v === 'function') {
    // do something
}
```
but unsure if the context matters.

Reply:
switched to the simpler version Jack mentioned, as underscore.js also uses that implementation